SConstruct

```diff
@@ -248,6 +248,8 @@ condition_map = {
     'POSIX_HOST' : env['PLATFORM'] == 'posix',
     'POWERPC_HOST' : False,
     'WINDOWS_HOST' : env['PLATFORM'] == 'win32',
+    'X86_HOST' : True,
+    'ZSERIES_HOST' : False,
```

Comment:
Sorry, I should have raised this point in a comment (I had a note to myself to re-visit this once the drop happened, but on second thought, that's obviously too late). Thanks for taking this on, @daveh86.